README.md

```diff
@@ -62,6 +62,80 @@ an easy way to setup bitbake based projects. For more details, visit the
 
 For a manual build without KAS, refer to the [Yocto Project Quick Build](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html).
 
+## Flash
+
+### Build QDL tool
+
+QDL tool communicates with USB devices of id `05c6:9008` and uploads a flash
+loader, which is then used for flashing images. Follow the steps below to
+download and compile QDL for your platform:
+
+1. Clone the QDL repository:
+
+   ```
+   git clone https://github.com/linux-msm/qdl
+   ```
+
+2. Read the README and install build dependencies (`libxml2-dev` and
+   `libusb-1.0-0-dev`). On Debian-based distribution run:
+
+   ```
+   sudo apt install libxml2-dev libusb-1.0-0-dev
+   ```
+
+3. Build the QDL tool using make:
+
+   ```
+   cd qdl
+   make
+   ```
+
+### Flash images
+
+Make sure that ModemManager is not running, disable it if necessary.
+Location of all DIP switches, USB debug port and buttons (`F_DL` for instance)
+can be found on [RB3 Gen 2 Quick Start Guide](https://docs.qualcomm.com/bundle/publicresource/topics/80-70014-253/ubuntu_host.html).
```

Comment:
I think having the generic instructions here is fine, but this layer is expected to support multiple machines (there is already support for ride), so should we isolate the board specific instructions and just have a link instead?

Reply:
For RIDE one needs to use alpaca to set the EDL mode. I don't think there are public instructions available for that.

Reply:
@ricardosalveti makes sense, will do

Reply:
I've tried to make flash instructions as much board-agnostic as possible. I've moved board specific details to a dedicated section.